File: 30 days of code/Day 01 - Data types/Data types.java
/**
 *              DATA TYPES | JAVA
 * Domain       30 DAYS OF CODE
 * Sub domain   DAY 01
 */

import java.io.File;
import java.io.FileNotFoundException;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) throws FileNotFoundException {
        File file = new File("input.txt");
        Scanner scan = new Scanner(file);
        {
            int i = Integer.parseInt(scan.nextLine());
            double d = Double.parseDouble(scan.nextLine());
            String s = scan.nextLine();

            System.out.println(4 + i);
            System.out.println(4.0 + d);
            System.out.println("HackerRank" + s);
        }
        scan.close();
    }
}

File: 30 days of code/Day 01 - Data types/Data types.cs
/**
 *              DATA TYPES | C#
 * Domain       30 DAYS OF CODE
 * Sub domain   DAY 01
 */

using System;
using System.Globalization;
using System.IO;

class Program
{
    static void Main(string[] args)
    {
        int i = 4;
        double d = 4.0;
        string s = "HackerRank ";

        var reader = new InputReader(false);
        int i2 = reader.NextInt;
        double d2 = reader.NextDouble;
        string s2 = reader.NextString;
        reader.Close();

        Console.WriteLine(i + i2);
        Console.WriteLine("{0:.0}", d + d2);
        Console.WriteLine(s + s2);
    }
}

class InputReader
{
    private TextReader input;

    public InputReader(bool fromFile)
    {
        input = fromFile ? new StreamReader("input.txt") : Console.In;
    }

    public string NextLine => input.ReadLine();

    public int NextInt => int.Parse(NextLine);

    public double NextDouble => double.Parse(NextLine, CultureInfo.InvariantCulture);

    public string NextString => NextLine;

    public void Close() => input.Close();
}
